src/main/java/net/fabricmc/loader/api/FabricLoader.java

```diff
@@ -183,4 +183,20 @@ static FabricLoader getInstance() {
 
 	@Deprecated
 	File getConfigDirectory();
+
+	/**
+	 * Gets the command line arguments used to launch the game. If this is printed for debugging, make sure {@code sanitize} is {@code true}.
+	 * @param sanitize Whether to remove sensitive information
+	 * @return the launch arguments
+	 */
+	String[] getLaunchArguments(boolean sanitize);
+
+	/**
+	 * @see FabricLoader#getLaunchArguments(boolean)
+	 * <p>
+	 * Does not contain any sensitive information.
+	 */
+	default String[] getLaunchArguments() {
```

Comment:
I'd remove this override, this is a niche API where allowing the caller to omit an arg doesn't provide any significant convenience benefit. It does however clutter the already large FabricLoader interface quite a bit more.

Reply:
This was suggested here: https://github.com/FabricMC/fabric-loader/pull/299#issuecomment-678670657

Reply:
I know, not happy with the suggestion.